### PR TITLE
Post

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added follow and unfollow
added home(feed) posts
added links to profile in post,follow suggest and comment
added individual profile

### CODE REVIEW

File: src/components/FollowSuggest.jsx [hunk 1]
```diff
@@ -12,9 +8,22 @@ import {
...
   HStack,
 } from "@chakra-ui/react";
-import {MdAdd} from "react-icons/md"
+import { MdAdd } from "react-icons/md";
+import { useSelector, useDispatch } from "react-redux";
+import { followUser } from "../features/auth/AuthSlice";
+import { useNavigate } from "react-router-dom";
 
 function FollowSuggest() {
+  // const [suggestUser, setSuggestUser] = useState([]);
```
Comment (Pratik1005) [inline src/components/FollowSuggest.jsx:17]: Remove commented code if not needed

File: src/features/auth/AuthSlice.js [hunk 2]
```diff
@@ -92,6 +158,36 @@ const authSlice = createSlice({
...
+    // [updateUser.pending]: (state) => {
+    //   console.log(state);
+    // },
+    // [updateUser.fulfilled]: (state, action) => {
+    //   state.user = action.payload;
+    //   localStorage.setItem("user", JSON.stringify(state.user));
+    // },
+    // [updateUser.rejected]: (state, action) => {
+    //   state.error = action.payload;
+    // },
```
Comment (Pratik1005) [inline src/features/auth/AuthSlice.js:170]: same here

File: src/features/user/UserSlice.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+
+const userSlice = createSlice({
+  name: "user",
+  initialState,
+  reducers: {},
+  extraReducers: {
+    [getAllUsers.pending]: (state) => {
+      // console.log(state);
```
Comment (Pratik1005) [inline src/features/user/UserSlice.js:28]: same here

**Review by Pratik1005 — APPROVED** [review]
Functionality working properly, good folder structure

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Classnames are self explanatory 
Good to merge

**Pull request merged**